### ISSUE TITLE
Remove editor in classic citation dialog

### ISSUE THREAD

**Issue by dstillman** [zotero/zotero#1462]
Is there any reason to keep it? If you really need to edit a citation (which you almost never should), you can just do so in the document (and you've been able to since Zotero 3.0, which is why the Quick Format dialog doesn't have an editor). The editor is almost never tested and ends up broken with some regularity.

**Comment by adomasven** [comment]
It doesn't trigger the "you've edited this citation" prompt (unless the citation style is changed).

**Comment by dstillman** [comment]
Yeah, but that's fine. It's still a legacy feature from before we added support for direct editing, so leaving it in but making people switch to the classic dialog doesn't really make sense.

**Comment by dstillman** [comment]
In other words, direct editing is what you're supposed to do these days if you really need to edit citations. The prompt is just the equivalent to the warning text we show below the editor in the classic dialog.

**Comment by adomasven** [comment]
Formatting changes (like italicisation) will not trigger the _citation modified_ prompt (since we're checking for plaintext mismatch).
